From: Andrew Morton <akpm@linux-foundation.org>
Subject: mm-rmap-use-huge_ptep_get-in-try_to_migrate_one-checkpatch-fixes
Date: Sun Jul  5 12:46:13 AM PDT 2026

ERROR: Please use git commit description style 'commit <12+ chars of sha1> ("<title line>")' - ie: 'Commit a98a2f0c8ce1 ("mm/rmap: split migration into its own function")'
#18: 
Commit a98a2f0c8ce1 copied the bug from try_to_unmap_one into

WARNING: braces {} are not necessary for any arm of this statement
#74: FILE: mm/rmap.c:2504:
+		if (folio_test_hugetlb(folio)) {
[...]
+		} else {
[...]

total: 1 errors, 1 warnings, 31 lines checked

NOTE: For some of the reported defects, checkpatch may be able to
      mechanically convert to the typical style using --fix or --fix-inplace.

./patches/mm-rmap-use-huge_ptep_get-in-try_to_migrate_one.patch has style problems, please review.

NOTE: If any of the errors are false positives, please report
      them to the maintainer, see CHECKPATCH in MAINTAINERS.

Please run checkpatch prior to sending patches

Cc: Dev Jain <dev.jain@arm.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/rmap.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/mm/rmap.c~mm-rmap-use-huge_ptep_get-in-try_to_migrate_one-checkpatch-fixes
+++ a/mm/rmap.c
@@ -2501,11 +2501,10 @@ static bool try_to_migrate_one(struct fo
 		VM_BUG_ON_FOLIO(!pvmw.pte, folio);
 
 		address = pvmw.address;
-		if (folio_test_hugetlb(folio)) {
+		if (folio_test_hugetlb(folio))
 			pteval = huge_ptep_get(mm, address, pvmw.pte);
-		} else {
+		else
 			pteval = ptep_get(pvmw.pte);
-		}
 		if (likely(pte_present(pteval))) {
 			pfn = pte_pfn(pteval);
 		} else {
_
